tools/src/benchmarker.cr

```diff
@@ -187,6 +193,26 @@ def puts_markdown(line, m_lines = nil, m = false)
   m_lines.push(line) if m && m_lines
 end
 
+####################
+### DEFAULT VALUES #
+####################
+threads = (System.cpu_count + 1).to_i
+requests = 10000
+record = false
+
+OptionParser.parse! do |parser|
```

Comment:
Doesn't this will crash when I run the benchmark like?
```
./bin/benchmarker --threads=8 kemal amber
```

Reply:
sure, any options

+ `./bin/benchmarker --threads=8 kemal amber÷`
+ `./bin/benchmarker kemal amber -t 8`
+ `./bin/benchmarker kemal amber --threads=8`

will lead to 
~~~crystal
threads = 8
all_frameworks = ['kemal','amber']
~~~

Reply:
awesome :+1: